numpy/compat/py3k.py

```diff
@@ -104,7 +101,15 @@ def is_pathlib_path(obj):
 
     Prefer using `isinstance(obj, os_PathLike)` instead of this function.
     """
-    return Path is not None and isinstance(obj, Path)
+    from pathlib import Path
+    # 2019-08-25, 1.18
+    warn("The use of is_pathlib_path is deprecated. "
+         "For Python 3.5, you should prefer using "
```

Comment:
In light of the new NEP, should this warning be revised? Should this whole PR be revised to just make `is_pathlib_like` the same as `os_PathLike`???

Reply:
According to https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst#drop-schedule

Should I simply remove references to Python 3.5?

Reply:
xref gh-14673